src/mongo/client/replica_set_monitor.h

```diff
@@ -124,6 +124,11 @@ class ReplicaSetMonitor {
     void appendInfo(BSONObjBuilder& b) const;
 
     /**
+     * Returns true if the monitor knows a usable primary from it's interal view.
```

Comment:
@kaloianm want to fix these in the server and I'll pick them up on next backport.
@samantharitter good catch